src/definitions_cxx.hpp

```diff
@@ -178,6 +178,7 @@ constexpr int32_t kScrollTime = 400;
 constexpr int32_t USE_DEFAULT_VELOCITY = 255;
 
 constexpr int32_t kMaxSequenceLength = 1610612736; // The biggest multiple of 3 which can fit in a signed 32-bit int32_t
+constexpr int32_t kMaxZoom = 100663296; // The zoom level in which the maximum sequence length is displayed on all pads
```

Comment:
can this be calculated rather than additional magic?

Reply:
That's how I had it before (`kMaxSequenceLength / 16`) but I thought maybe it was a bad idea to calculate this on every zoom. I don't really now what is and isn't a big deal when it comes to performance...

Reply:
`constexpr` expressions are calculated at compile time, so that should be no issue (well, technically the compiler is allowed to not but this is trivial constant propagation so it will in practice)

Reply:
Ohh... I get what you're saying. I interpreted the feedback wrong initially.